css/styles.css

```diff
@@ -57,3 +57,56 @@ textarea {
   font-weight: bold;
   letter-spacing: 0.05rem;
 }
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.nav-style {
+  padding-left: 2rem;
+  text-decoration: none;
+  font-size: 1.3rem;
+  background-color: #562b81;
+  border: none;
+}
+
+.align {
+  margin-left: 25rem;
+}
+
+.new-entry-button {
+  color: white;
+  background-color: #562b81;
+  padding: 0.3rem 1rem;
+  border-radius: 0.4rem;
+  text-decoration: none;
+  border: none;
+}
+
+.entries-style {
+  font-size: 1.8rem;
+  font-weight: bolder;
+}
+
+.hidden {
+  display: none;
+}
+
+.visible {
+  display: visible;
+}
```

Comment:
this is not valid CSS, it isn't doing anything

Reply:
fixed